test/test_rack_handler.rb

```diff
@@ -2,16 +2,22 @@
 
 require "rack"
 
-if Rack::RELEASE < '3'
-
-  require "rack/handler/puma"
+module TestRackUp
+  if Rack::RELEASE < '3'
+    require "rack/handler/puma"
```

Comment:
this can be moved out of the if statement because you use it in both branches

Reply:
Updated, see commit 'Changes per comments'